### ISSUE TITLE
Multiple cli_root/zfs_destroy tests fail due to failing newfs command

### ISSUE THREAD

**Issue by ZackaryWelch** [zfsonfreebsd/ZoF#66]
Tests in cli_root/zfs_destroy, namely zfs_destroy_001, 005, 008, 009, 010, 011, 012, and 013, fail with the error:
```
newfs /dev/zvol/testpool/testctr/testvol > /dev/null
```
generated in zfs_destroy_common.kshlib with the command not returning 0:
```
echo "y" | /sbin/newfs $ZVOL_DEVDIR/$VOL > /dev/null 2>&1
```

**Comment by ZackaryWelch** [comment]
Related to https://github.com/zfsonfreebsd/ZoF/issues/69 and https://github.com/zfsonfreebsd/ZoF/issues/68

**Comment by ZackaryWelch** [comment]
Closing as I've merged this issue with #68
